RecoTracker/IterativeTracking/python/iterativeTk_cff.py

```diff
@@ -43,6 +43,25 @@
     InitialStepPreSplitting +
     InitialStep +
     HighPtTripletStep +
+    DetachedQuadStep +
```

Comment:
is there a good reason for having DetachedQuadStep before the LowPt\*Step in trackingPhase1 and after them in trackingPhase1PU70?
there would be less changes to make.